### ISSUE TITLE
Example in a docker container

### ISSUE THREAD

**Issue by jleni** [crowdAI/marLo#19]
Very nice seeing marLo getting ready.

At the moment, it is very difficult to get some basic examples to work in a clean machine. I could get a few to run (single_agent), but chainer_test_* are not always working because enviroments are not registered, etc.

While documentation is appreciated, I think a few minimal Dockerfile + run.sh script would be much more convenient. Most people will be coding and will find easier to use that as a reference. It can also be sued for regression testing. The purpose is not to use the container but to provide a 100% reproducible environment to use a guide.

**Comment by spMohanty** [comment]
@jleni :  good point. Please note that at the moment, the goal is to get just the singleagent and multiagent examples working.
We will be updating all the chainer examples soon.

And sure, we will add a dockerfile soon to help reproduce the environment. It is anyway scheduled to be a part of the submissions instructions scheduled to be released in a week.
